tests/main.test.js

```diff
@@ -69,14 +69,14 @@ test("Subdirectories", async ({ page, prefix, baseUrl }) => {
 	await expect(await page.getByText("Docs").getAttribute("href")).toEqual(prefix + "/docs/");
 	await expect(await page.getByText("Homepage").getAttribute("href")).toEqual(prefix || "/");
 
-	await page.getByText("b").click();
+	await page.getByText("b", { exact: true }).click();
```

Comment:
i can understand why this change.. will query inside [baseElement](https://testing-library.com/docs/react-testing-library/api#baseelement) but you use `page` objest so exact: true i guess is a way to dont choose another different thant the exact href to test? i dont know about such parameter for